.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  php-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.4, 7.3, 7.2]
+        laravel: [7.*]
+        testbench: [5.*]
+        dependency-version: [prefer-lowest, prefer-stable]
+        include:
+          - laravel: 7.*
+            testbench: 5.*
```

Comment:
I'm wonder why this is necessary?

Got them working quite nicely without having to specify this , see https://github.com/mfn/laravel-ide-helper/actions/runs/143792113 ( I was already working on PR but now stopped in favor of yours, joining forces 💪 )

Reply:
Hmm I also though it was ugly, but remembered having problems with it before. But PR away :)